l1-contracts/deploy-scripts/upgrade/EcosystemUpgrade.s.sol

```diff
@@ -1348,10 +1367,19 @@ contract EcosystemUpgrade is Script, DeployL1Script {
     ) public virtual returns (Call[] memory calls) {
         bytes memory l2Calldata = abi.encodeCall(
             RollupDAManager.updateDAPair,
-            (gatewayConfig.gatewayStateTransition.rollupDAManager, getExpectedL2Address("RollupL2DAValidator"), true)
+            (
+                gatewayConfig.gatewayStateTransition.rollupL2DAValidator,
```

Comment:
please rename to `rollupSLDAValidator`, because right now there is a collision in names with `RollupL2DAValidator` (as you can see below you ask for `getExpectedL2Address("RollupL2DAValidator")` and here you are using `rollupL2DAValidator`)